packages/payment-detection/src/erc20/address-based-info-retriever.ts

```diff
@@ -87,19 +85,19 @@ export default class ERC20InfoRetriever
     const logs = await provider.getLogs(filter);
 
     // Clean up the Transfer logs data
-    const eventPromises = logs.map(async log => {
+    const eventPromises = logs.map(async (log) => {
       if (!log.blockNumber) {
         throw new Error('Block number not found');
       }
       const block = await provider.getBlock(log.blockNumber);
       const parsedLog = contract.interface.parseLog(log);
       return {
-        amount: parsedLog.values.value.toString(),
+        amount: parsedLog.args.value.toString(),
         name: this.eventName,
         parameters: {
           block: block.number,
-          from: parsedLog.values.from,
-          to: parsedLog.values.to,
+          from: parsedLog.args.from,
+          to: parsedLog.args.to,
```

Comment:
very strange... for this contract, the `args` object contains the values by keys... the utility I added isn't useful here. I've tried importing the ABI in the same way it's done for this contract, but doesn't change anything...